flows/vision-zero/make_kv_store.py

```diff
@@ -17,6 +17,14 @@
     "AWS_CSV_ARCHIVE_BUCKET_NAME": os.getenv("AWS_CSV_ARCHIVE_BUCKET_NAME"),
```

Comment:
This file was moved up to the project level folder as it is intended to create a KV store for _all_ VZ flows, not just one.